doc/source/quick_start.md

```diff
@@ -14,253 +14,81 @@ kernelspec:
 
 # Quick start
 
-The following presents a descriptive example show-casing all core aspects of GeoUtils.
+A short code example using several end-user functionalities of GeoUtils. For a more detailed example of all features,
+have a look at the {ref}`feature-overview` page! Or, to find an example about
+a specific functionality, jump to {ref}`quick-gallery` right below.
 
-For more details, refer to the {ref}`core-index`, {ref}`rasters-index` or {ref}`vectors-index` pages.
+## Short example
 
-To find an example about a specific functionality, jump to {ref}`quick-gallery`.
+```{note}
+:class: margin
+
+Most functions examplified here normally require many lines of code using several independent packages with inconsistent
+geospatial syntax and volatile passing of metadata that can lead to errors!
+
+In GeoUtils, **these higher-level operations are tested to ensure robustness and consistency**. 🙂
+```
 
-```{tip}
-All pages of this documentation containing code cells can be **run interactively online without the need of setting up your own environment**. Simply click the top launch button!
+The package functionalities revolve around the
+{class}`~geoutils.Raster` and {class}`~geoutils.Vector` classes, from which most methods can be called.
+Below, in a few lines, we load a raster and a vector, crop them to a common extent, re-assign raster values around
+a buffer of the vector, perform calculations on the modified raster, and finally plot and save it!
 
-Alternatively, start your own notebook to test GeoUtils at [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/GlacioHack/geoutils/main).
+<!--- An empty margin to add some vertical padding --->
+```{margin}
+&nbsp;
 ```
 
-## The core {class}`~geoutils.Raster` and {class}`~geoutils.Vector` classes
 
-In GeoUtils, geospatial handling is object-based and revolves around {class}`~geoutils.Raster` and {class}`~geoutils.Vector`.
-These link to either **on-disk** or **in-memory** datasets, opened by instantiating the class.
+```{note}
+:class: margin
+
+**We notice a ``Userwarning``:** No nodata value was defined in the GeoTIFF file, so GeoUtils automatically defined
+one compatible with the data type to use during operations.
+```
 
 ```{code-cell} ipython3
```

Comment:
Will these kind of code cells be tested during CI?
The outdated command in the comparison geoutils vs rasterio made me think that we should avoid code cells that are not tested.

Reply:
Unfortunately we cannot nest code cells right now, maybe in the future: https://github.com/executablebooks/jupyter-book/issues/1178
We could still take the code from somewhere else (a Python file) that is executed independently during tests to check the code in up-to-date, like we did before.
I'll do that

Reply:
Took a bit of effort, but done!
There were actually a lot more errors than just the `dst_`... :sweat_smile: